### PR TITLE
TV by the Numbers.js -- fix eslint warnings

### PR DESCRIPTION
**Pull request by retorquere**

### CODE REVIEW

**Comment by retorquere** [comment]
updated for @zotero/eslint-config@1.0.2

File: TV by the Numbers.js [hunk 1]
```diff
@@ -36,15 +36,14 @@
...
+	else if (doc.body.className.includes("single-post")) return "blogPost";
+	return false;
 }
 
 function doWeb(doc, url) {
 	if (detectWeb(doc, url) == "multiple") {
-		Zotero.selectItems(getSearchResults(doc, false), function(items) {
+		Zotero.selectItems(getSearchResults(doc, false), function (items) {
 			if (!items) {
 				return true;
```
Comment (zuphilip) [inline TV by the Numbers.js]: return;

File: TV by the Numbers.js [hunk 2]
```diff
@@ -53,8 +52,10 @@ function doWeb(doc, url) {
 				articles.push(i);
 			}
 			ZU.processDocuments(articles, scrape);
+			return true;
```
Comment (zuphilip) [inline TV by the Numbers.js]: no

**Comment by zuphilip** [comment]
Thank you!

**Pull request merged**